src/index.ts

```
@@ -165,6 +165,10 @@ async function startServer() {
         await import("./jobs/investmentWithdrawalJob");
       await startInvestmentWithdrawalScheduler();
 
+      // Start yield accrual scheduler (run once at startup to seed accruals)
+      const { startYieldAccrualScheduler } = await import("./jobs/yieldAccrualJob");
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)